### PR TITLE
Components created

### PR DESCRIPTION
**Pull request by amudric**

### CODE REVIEW

File: izadji/src/components/Team/Team.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            </div>
+          </div>
+
+          {/* biography */}
+          <TeamMemberBio
+            name={selectedTeamMember.name}
+            jobTitle={selectedTeamMember.jobTitle}
+            profileImage={selectedTeamMember.profileImage}
+            bio={selectedTeamMember.bio}
+          ></TeamMemberBio>
```
Comment (LazarJ98) [inline izadji/src/components/Team/Team.tsx]: For component without children, we do not need to use both opening and ending tag. So on lines 63-68, you could just put it like this:

```
          <TeamMemberBio
            name={selectedTeamMember.name}
            jobTitle={selectedTeamMember.jobTitle}
            profileImage={selectedTeamMember.profileImage}
            bio={selectedTeamMember.bio}
          />
```

File: izadji/src/components/TeamMemberItem/TeamMemberItem.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import styles from './TeamMemberItem.module.scss';
+
+export type TeamMemberItemProps = {
+  name: string;
+  jobTitle: string;
+  profileImage: string;
+  email: string;
+  bio: string;
```
Comment (LazarJ98) [inline izadji/src/components/TeamMemberItem/TeamMemberItem.tsx:8]: It seems to me that we do not use **bio** prop anywhere inside the TeamMemberItem component. Maybe we can delete it?

Comment (amudric) [reply izadji/src/components/TeamMemberItem/TeamMemberItem.tsx:8]: :)

**Review by LazarJ98 — APPROVED** [review]

File: izadji/src/components/TeamMemberBio/TeamMemberBio.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <div className={styles.people__slider__bio__item__container}>
+            <div className={styles.people__slider__item__bio__holder}>
+              <img
+                className={`${styles.people__slider__bio__img} ${styles.has__cover}`}
+                src={profileImage}
+                alt={name}
+              />
+            </div>
+            <div className={styles.people__slider__text__bio__holder}>
+              <span className={`${styles.people__slider__bio__text__bold}`}>
```
Comment (DusicaVasicFax) [inline izadji/src/components/TeamMemberBio/TeamMemberBio.tsx]: We don't need template literals here because we only have one style class :)

**Pull request merged**